docs/ota-client-guide/modules/ROOT/pages/reporting-problems.adoc

```diff
@@ -0,0 +1,60 @@
+= Reporting problems
+ifdef::env-github[]
+
+[NOTE]
+====
+We recommend that you link:https://docs.ota.here.com/ota-client/latest/{docname}.html[view this article in our documentation portal]. Not all of our articles render correctly in GitHub.
+====
+endif::[]
+
+:page-layout: page
+:page-categories: [tips]
+:icons: font
+
+== Contacting HERE OTA Connect support
+
+If you encounter a problem with {product-name} and the xref:troubleshooting.adoc[Troubleshooting article] doesn't address it, please contact link:mailto:otaconnect.support@here.com[otaconnect.support@here.com] and we'll do our best to help! To help us diagnose your issue, please provide us with as much information as possible. What follows are some useful artifacts as well as how to retrieve them.
+
+=== aktualizr logs with debug logging
+
+By default, aktualizr writes logs to the systemd logger, which you can read with `journalctl -u aktualizr`. If you run aktualizr manually on the commandline, the logging is printed on stdout. The default loglevel is 2 (info). Lower loglevels are more verbose. To change the loglevel, there are two options:
+
+* To build a new image with debug logging enabled, add `IMAGE_INSTALL_append = " aktualizr-log-debug "` to your `local.conf` and bitbake a new image. See the xref:meta-updater-usage.adoc#_aktualizr_configuration[meta-updater usage article] for more details.
+* To enable debug logging on a device accessible via the commandline without building a new image, stop the aktualizr service on the device (`systemctl stop aktualizr`) and manually start aktualizr on the commandline, e.g. `aktualizr --loglevel 0`.
+
+See also the xref:aktualizr-config-options.adoc#_logger[aktualizr configuration options article].
+
+=== Output of aktualizr-info
+
+See the xref:debugging-tips.adoc#_inspect_stored_info_with_aktualizr_info[debugging tips] for more information.
+
+=== aktualizr version
+
+On an active device accessible via the commandline, enter `aktualizr --version`. Otherwise, you can examine the link:https://github.com/advancedtelematic/meta-updater/blob/master/recipes-sota/aktualizr/aktualizr_git.bb[aktualizr recipe in meta-updater] in your build environment and look for the `SRCREV` variable.
```

Comment:
```suggestion
On an active device accessible via the commandline, enter `aktualizr --version`. Otherwise, you can examine the aktualizr recipe in meta-updater in your build environment and look for the `SRCREV` variable. This recipe will normally be located (relative to your Yocto build directory) at `../meta-updater/recipes-sota/aktualizr/aktualizr_git.bb`.
```
As much as I'd like to hope that people will follow this instruction correctly, I strongly suspect that they'll follow your link to the latest master version of `aktualizr_git.bb` and report the version there, instead of looking at their local copy.